### PR TITLE
Add arista eos show ip helper

### PR DESCRIPTION
**Pull request by targuan**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT
arista_eos_show_ip_helper-address.template
OS: arista EOS
command: show ip helper-address

##### SUMMARY
This template allow the collection of the IP helpers from ip interfaces.

### CODE REVIEW

File: templates/arista_eos_show_ip_helper-address.template [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Value Required INTERFACE (\S+)
+Value List IP_HELPER (\d+\.\d+\.\d+\.\d+|\S+)
+
+Start
+ ^\S -> Continue.Record
```
Comment (FragmentedPacket) [inline templates/arista_eos_show_ip_helper-address.template]: Can you add `^DHCP` above L5 and `^\s+DHCP\s+Smart` above L7? And then add the following under L8?
```
 ^\s*$$
 ^. -> Error
```

File: tests/test_index_load.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+try:
```
Comment (FragmentedPacket) [inline tests/test_index_load.py]: Do you mind removing this? We would want this as a separate PR

Comment (targuan) [reply tests/test_index_load.py]: This was just a test, it shouldn't have been there.

File: templates/arista_eos_show_ip_helper-address.template (hunk 1 above)
Comment (targuan) [reply templates/arista_eos_show_ip_helper-address.template]: This has been corrected

File: templates/arista_eos_show_ip_helper-address.template [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Value Required INTERFACE (\S+)
+Value List IP_HELPER (\d+\.\d+\.\d+\.\d+|\S+)
+
+Start
+ ^DHCP
+ ^\S -> Continue.Record
```
Comment (FragmentedPacket) [inline templates/arista_eos_show_ip_helper-address.template]: Can you change L6 to `^Interface -> Continue.Record`?

Once that is changed, I will merge it.

Thanks for your work on the template!

Comment (targuan) [reply templates/arista_eos_show_ip_helper-address.template]: Uclvri kp LS

**Pull request merged**